app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
Good find. Where are those kwargs getting used though / where is the user's role getting set to that role?